### PR TITLE
Add AltDrag version 1.1

### PR DESCRIPTION
**Pull request by Calinou**
This was requested by @stuaxo in https://github.com/lukesampson/scoop/issues/3054.

This looks pretty interesting (and it works without administrator privileges), so I might use it too :slightly_smiling_face:

### CODE REVIEW

File: bucket/altdrag.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+    "homepage": "https://github.com/stefansundin/altdrag",
```
Comment (Ash258) [inline bucket/altdrag.json]: io homepage would be better

```suggestion
    "homepage": "https://stefansundin.github.io/altdrag/",
```

Comment (Ash258) [reply bucket/altdrag.json]: Change checkver after that

File: bucket/altdrag.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    "shortcuts": [
+        [
+            "AltDrag.exe",
+            "AltDrag"
+        ]
+    ],
+    "persist": "AltDrag.ini",
+    "checkver": "github",
+    "autoupdate": {
+        "url": "https://github.com/stefansundin/altdrag/releases/download/v$version/AltDrag-$version.zip"
```
Comment (Ash258) [inline bucket/altdrag.json]: Add hash extraction

```suggestion
        "url": "https://github.com/stefansundin/altdrag/releases/download/v$version/AltDrag-$version.zip",
        "hash": "$baseurl/sha256sum.txt"
```

File: bucket/altdrag.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
+{
+    "homepage": "https://github.com/stefansundin/altdrag",
+    "description": "Easily drag windows when pressing the Alt key",
+    "license": "GPL-3.0-or-later",
+    "version": "1.1",
+    "url": "https://github.com/stefansundin/altdrag/releases/download/v1.1/AltDrag-1.1.zip",
+    "hash": "5e1cf4fd8bfbdeca672cd53141019471b344317c81fdefe1ae9cb3f96183bdf9",
+    "extract_dir": "AltDrag",
```
Comment (Ash258) [inline bucket/altdrag.json]:
```suggestion
    "extract_dir": "altdrag-1.1",
```

Folder contains version number. Update autoupdate accordingly

zip

Comment (Ash258) [reply bucket/altdrag.json]: There is no exe inside zip tho.

Zip

Comment (Ash258) [reply bucket/altdrag.json]: Sorry, my bad

**Comment by stuaxo** [comment]
This along with XMouse and I almost feel at home.

**Comment by stuaxo** [comment]
Thanks :)

**Pull request merged**